lib/server.js

```diff
@@ -0,0 +1,11 @@
+import app from './services/app';
+
+const PORT = 3000;
```

Comment:
pleasse create configuration file and move all such variables to them

Reply:
[Resolved](https://github.com/inst-event-emitter/eventemitter-backend/pull/1/commits/2ebc07740b45973e3a84e5c2f649005a9b97b061)